include/open62541/plugin/eventloop.h

```diff
@@ -454,12 +454,17 @@ struct UA_InterruptManager {
 #if defined(UA_ARCHITECTURE_POSIX) || defined(UA_ARCHITECTURE_WIN32)
 
 /**
- * POSIX EventLop Implementation
+ * POSIX EventLoop Implementation
  * -----------------------------
  * The POSIX compatibility of Win32 is 'close enough'. So a joint implementation
  * is provided. The configuration paramaters must be set before starting the
  * EventLoop.
  *
+ * **Socket configuration**
+ *
+ * 0:max-sockets [uint32]
```

Comment:
This is now even "more global".
You added the limit
- not to a specific server socket
- not to the TCP subsystem
but to the entire EventLoop.

In that case we need to also ensure that we do not exceed the number of sockets across TCP and UDP!